### PR TITLE
made distributed-performance project card

### LINKED ISSUE #482: Create new project card for "Service Mesh Distributed Performance Management"
**Current Situation**

@kushthedude is working through a Google Summer of Code internship that is focused on incorporating Envoy proxy's Nighthawk load generator into Meshery. We are missing a [project page](https://layer5.io/projects) that describes this.

**Implementation**

- This is the start to a project page (it began as a copy of the SMI project page) - 
https://github.com/layer5io/layer5/blob/master/collections/_projects/service-mesh-distributed-performance-management.md.
- a new project card should show up here https://layer5.io/projects

**Resources**
- this blog post describes the project - https://layer5.io/blog/gsoc-with-layer5-and-cncf
- this proposal describes the project - https://docs.google.com/document/d/1qgvkTYSBDga-QmLHR9uK-utgiWAUcpF5-2Z4zaNqej0/edit?usp=sharing
- this design spec describes the project - https://docs.google.com/document/d/1_hhQ5w1iLClgf7RvboI6il-eMKKAVPwz50GHm2VN4Bg/edit?usp=sharing

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jashpatel1 <patel.5@iitj.ac.in>

**Description**

This PR fixes #482 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_pages/service-mesh-distributed-performance-management.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+layout: project
+title:  "Distributed Performance Management of Service Meshes"
+description: "Layer5 is an industry leader in the management of service meshes and their workloads.
+In collaboration with Google, Layer5 is advancing the state of the industry through novel projects like that of distributed performance management."
+author:
+  twitter: layer5
+image:
+  path: /assets/images/buttons/load.png
+  width: 1200
+thumbnail: /assets/images/buttons/load.png
```
Comment (leecalcote) [inline collections/_pages/service-mesh-distributed-performance-management.html:11]: @Jashpatel1 whoa. Nice job on the project icon!

File: collections/_projects/service-mesh-distributed-performance-management.md [hunk 2]
```diff
@@ -6,53 +6,40 @@ In collaboration with Google, Layer5 is advancing the state of the industry thro
...
-    </ul>
-    <h4>Validating Conformance</h4>
-<p>Conformance to SMI specifications will be done through use of a service mesh’s workload. A sample application is used as the workload to test.</p>
-<h5>Sample Application</h5>
-<p>To facilitate a common set of tests, a sample application has been developed for purposes of providing a consistent workload to apply SMI specs against. 
-  A deployment of the <a href="https://github.com/layer5io/learn-layer5">Learn Layer5</a> sample application being fitted to each service mesh.</p>
+    <p>Currently, Meshery which is a multi-service mesh management plane which can operate & adapt to various different service meshes only supports a single way load testing. Understanding the need to support a variety of performance testing scenarios, Meshery has been equipped with a load generator interface as a point of extensibility to support the use of different load generators (see <a href="https://meshery.layer5.io/docs/extensibility">Meshery Extensibility</a> documentation). Meshery Load Generation currently has been equipped with support for both wrk2 and fortio, neither of which support distributed load testing.</p>
```
Comment (kushthedude) [inline collections/_projects/service-mesh-distributed-performance-management.md:37]: @Jashpatel1 Why am I seeing the same file added above in _pages section ?

File: _data/navigation.yaml [hunk 3]
```diff
@@ -14,8 +14,12 @@
       new_window: false
     - name: Service Mesh Interface Conformance
       link: /service-mesh-interface
-      new_window: true    
-      external: true        
+      new_window: false
```
Comment (kushthedude) [inline _data/navigation.yaml:17]: Why the following change?

File: collections/_projects/service-mesh-distributed-performance-management.md [hunk 4]
```diff
@@ -6,53 +6,40 @@ In collaboration with Google, Layer5 is advancing the state of the industry thro
...
 
 ---
 <div class="row" id="smi" >
-  <div class="col m6">
+  <div class="col m8">
     <h3>
       Purpose and Overview
     </h3>
-    <p>The scope of this initiative includes all service mesh projects participating in the Service Mesh Interface specification. It’s important to acknowledge that conformance consists of both capabilities and compliance status. 
+    <p>Distributed load testing offers insight into system behaviors that arguably more accurately represent real world behaviors of services under load as that load comes from any number of sources. This project is being developed by Kush Trivedi as a part of Google Summer of Code intern with CNCF and Layer5.
```
Comment (kushthedude) [inline collections/_projects/service-mesh-distributed-performance-management.md:23]: I see the same page being added above in the PR under _pages folder, why?

File: _data/navigation.yaml (hunk 3 above)
Comment (leecalcote) [reply _data/navigation.yaml:17]: @kushthedude The new project page is internal to the layer5.io site, so we will have this link open in the _parent, not _blank target.

File: collections/_projects/service-mesh-distributed-performance-management.md (hunk 4 above)
Comment (leecalcote) [reply collections/_projects/service-mesh-distributed-performance-management.md:23]: @kushthedude good eye. This is part of the current site design that is something of a work around for the way that Jekyll and GitHub Pages handle trailing slashes on the end of URLs. I'm hopeful we can be right of the duplication at some point.

**Pull request merged**